src/OrchardCore.Modules/OrchardCore.Users/TimeZone/Services/UserTimeZoneService.cs

```diff
@@ -22,17 +26,17 @@ public class UserTimeZoneService
     public UserTimeZoneService(
         IClock clock,
         IDistributedCache distributedCache,
-        IHttpContextAccessor httpContextAccessor
-    )
+        IHttpContextAccessor httpContextAccessor)
```

Comment:
This is not needed and should be removed

Reply:
Why?